readthedocs/projects/tasks.py

```diff
@@ -1768,3 +1789,32 @@ def retry_domain_verification(domain_pk):
         sender=domain.__class__,
         domain=domain,
     )
+
+
+@app.task(queue='web')
+def send_build_status(build_pk, state):
+    """
+    Send Build Status to Git Status API for project external versions.
+
+    :param build_pk: Build pk
+    :param state: build state failed, pending, or success to be sent.
+    """
+    build = Build.objects.get(pk=build_pk)
+    version = build.version
+    project = build.project
+
+    # Send status reports for only External (pull/merge request) Versions.
+    if version.type != EXTERNAL:
+        return
+
+    if 'github.com' in project.repo:
```

Comment:
I think you can get this from `project.remote_repository.account.provider`

Reply:
Thanks! I did not know this :100: